pkg/lint/runner.go

```diff
@@ -48,8 +48,10 @@ func NewRunner(astCache *astcache.Cache, cfg *config.Config, log logutils.Log, g
 		return nil, err
 	}
 
-	skipDirs := append([]string{}, packages.StdExcludeDirRegexps...)
-	skipDirs = append(skipDirs, cfg.Run.SkipDirs...)
+	skipDirs := cfg.Run.SkipDirs
+	if cfg.Run.UseDefaultSkipDirs {
+		skipDirs = append([]string{}, packages.StdExcludeDirRegexps...)
```

Comment:
Hey @Sean-Der, great to meet you  :)

Does it mean that `skip-dirs-use-default: true` (which will be true by default) will replace the explicit `skip-dirs` setting?
```go
	skipDirs := cfg.Run.SkipDirs // ["e2e", "foo/bar"]
	if cfg.Run.UseDefaultSkipDirs {
		skipDirs = append([]string{}, packages.StdExcludeDirRegexps...)
	}
	skipDirs // is now equal to packages.StdExcludeDirRegexps
```
Probably we want the `skipDirs = append(skipDirs, packages.StdExcludeDirRegexps...)` here?